scripts/lint-style.lean

```diff
@@ -91,6 +91,30 @@ def getLakefileLeanOptions : IO Lean.Options := do
       #[]
   return toLeanOptions (rootOpts.appendArray defaultOpts)
 
+/-- Compute the transitive closure of imports starting from a given file.
```

Comment:
Can you move this to a better location? `TextBased.lean` would already be slightly better (as other files in mathlib could import it), but a new utility file with both `findImports` and this function would be even better? (Or can/should this use `ImportGraph` instead?)